### DISCUSSION TITLE
How to get an image of painted UI?

### DISCUSSION THREAD

**Discussion by BctfN0HUK7Yg** [emilk/egui#1017] [Q&A]
Is it possible and how to?
I have several tabs with generated Ui and want to use a preview image of them but without new computation. I want to paint ui once and get the image form output

**Comment by emilk** [comment]
Getting an image for it is outside the scope of `egui`, but maybe with some work one could make `egui` spit out the `Shape`:s that a panel, window etc generates. Check out `GraphicLayers` if you are interested in working on this!
